docs/osx.md

```diff
@@ -11,7 +11,7 @@ and paste **a single line at a time** and press the `Enter` key afterwards. We s
 
 If something fails (and we hope it does not) troubleshooting will be much easier if you can share that output when asking for help.
 
-Happy deploying!
+Happy deploying!!
```

Comment:
Great. Good testing, now let's return this to a single `!`

Reply:
Ok I fixed this